Parameters were being propagated through celery tasks into `requests.request` method causing errors.
 